src/components/Grid/GridRow.tsx

```diff
@@ -1,28 +1,77 @@
 import styled from 'styled-components';
 import { MOBILE_ONLY_MEDIA_QUERY, TABLET_MEDIA_QUERY, DESKTOP_MEDIA_QUERY } from '../../styles';
 
-export const GridRow = styled('div')`
-  display: grid;
+type mobileValues = 1 | 2 | 3 | 4;
+type tabletDesktopValues = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+interface Props {
+  desktop?: boolean | tabletDesktopValues;
+  tablet?: boolean | tabletDesktopValues;
+  mobile?: boolean | mobileValues;
+}
+
+export const GridRow = styled.div<Props>`
+  display: none;
 
   @media ${MOBILE_ONLY_MEDIA_QUERY} {
     grid-template-columns: repeat(4, minmax(50px, 1fr));
     grid-column-gap: 24px;
     min-width: 272px;
     margin-left: 24px;
     margin-right: 24px;
+
+    ${({ mobile }) =>
+      mobile && typeof mobile === 'number'
```

Comment:
I'm assuming this will never be undefined so you can just cast 'mobile' to a number by doing +mobile